desktop-environment.el

```diff
@@ -152,6 +152,10 @@ The shell command should let the user interactively select the
 portion of the screen."
   :type 'string)
 
+(defcustom desktop-environment-screenshot-delay-flag " -d %d"
```

Comment:
flag -> argument

Reply:
should be resolved as of a4c484544a5409740d161ed506d7aa3376013817